Packer/ArkCase-CE/packer.json

```diff
@@ -16,7 +16,7 @@
       "source_ami": "{{ user `base_ami` }}",
       "instance_type": "t3.xlarge",
       "ssh_username": "centos",
-      "ami_name": "ArkCase CE Core {{ user `version` }} [{{isotime \"2006-01-02T15:04:05\"}}]",
+      "ami_name": "ArkCase CE Core {{ user `version` }} [{{isotime \"2006-01-02T15:04:05\"}} | clean_resource_name]",
```

Comment:
Shouldn't the `| clean_resource_name` be inside the `}}`?

Reply:
Yes :), I've already corrected -> https://github.com/ArkCase/DevOps/pull/9